### PR TITLE
[feat] Homepage

### PR DESCRIPTION
**Pull request by shivampandey0**
Added Following to Homepage
- Logo
- Navbar
  - search
  - login
  - wishlist
  - cart
- Landing hero image
- Featured Categories
- Footer

image

### CODE REVIEW

**Review by raghav2110 — APPROVED** [review]
The website looks great , also the code looks real clean and readable. Good to merge

File: css/base.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+html {
+  /* outline: 1px solid black; */
```
Comment (ZuberDunge) [inline css/base.css:2]: you can remove commented code

Comment (shivampandey0) [reply css/base.css:2]: that's for development use only. 
will be removed in the end.

**Review by ZuberDunge — APPROVED** [review]
Clean and clear code, also, I liked the naming convention you followed, Good work Shivam.

**Pull request merged**